src/main/java/com/hpe/adm/octane/ideplugins/services/EntityService.java

```diff
@@ -51,23 +52,33 @@
     private OctaneVersionService versionService;
 
     public Collection<EntityModel> findEntities(Entity entity) {
-        return findEntities(entity, null, null, null, null, null, null, null);
+        return findEntities(entity, null, null, null, null, null, null, null, null);
     }
 
     public Collection<EntityModel> findEntities(Entity entity, Query.QueryBuilder query, Set<String> fields) {
-        return findEntities(entity, query, fields, null, null, null, null, null);
+        return findEntities(entity, query, fields, null, null, null, null, null, null);
     }
 
     public Collection<EntityModel> findEntities(Entity entity, Query.QueryBuilder query, Set<String> fields, Map<String, Set<String>> expand) {
-        return findEntities(entity, query, fields, expand, null, null, null, null);
+        return findEntities(entity, query, fields, expand, null, null, null, null, null);
+    }
+
+    public Collection<EntityModel> findEntities(Entity entity, Query.QueryBuilder query, Set<String> fields, Boolean orderByUserItem) {
+        return findEntities(entity, query, fields, null, null, null, null, null, orderByUserItem);
     }
 
     public Collection<EntityModel> findEntities(Entity entity, Query.QueryBuilder query, Set<String> fields, Map<String, Set<String>> expand, Integer offset, Integer limit) {
-        return findEntities(entity, query, fields, expand, offset, limit, null, null);
+        return findEntities(entity, query, fields, expand, offset, limit, null, null, null);
     }
 
-    public Collection<EntityModel> findEntities(Entity entity, Query.QueryBuilder query, Set<String> fields, Map<String, Set<String>> expand, Integer offset, Integer limit, String orderByField, Boolean orderByAsc) {
-        EntityList entityList = octaneProvider.getOctane().entityList(entity.getApiEntityName());
+    public Collection<EntityModel> findEntities(Entity entity, Query.QueryBuilder query, Set<String> fields, Map<String, Set<String>> expand, Integer offset, Integer limit, String orderByField, Boolean orderByAsc, Boolean orderByUserItem) {
+        EntityList entityList;
+
+        if (orderByUserItem != null && orderByUserItem) {
+            entityList = octaneProvider.getOctane().entityList(entity.getApiEntityName() + "/order_by_user_item");
```

Comment:
I know we have discussed this is done because appending something to the URL is not supoported by the Java SDK version that the plugin is using and changing it would require a lot of rework - please add feature to the backlog for upgrading the SDK and a todo here to do it properly when the new SDK is in place